doc/examples/index.rst

```diff
@@ -7,4 +7,5 @@ Examples
 
     train_mlp_locally
     train_classifier_sagemaker
-    nlp_finetuning
\ No newline at end of file
+    nlp_finetuning
+    functional_usage
```

Comment:
I see the same problem I mentioned for the title

Reply:
Renamed.